src/core/type.ml

```diff
@@ -2824,6 +2824,10 @@ module TExprToExpr = struct
 end
 
 module ExtType = struct
+	let is_mono = function
+		| TMono _ -> true
```

Comment:
Since it's a general purpose function, I guess the intent is to check if it's an unbound monomorph here? Then it should be `TMono { contents = None }`

Reply:
fixed